### DISCUSSION TITLE
Nextcloud AIO  - nextcloud release

### DISCUSSION THREAD

**Discussion by agadinmd** [nextcloud/all-in-one#653] [Questions] [answered]
Hello,

how to findout the roadmap for nexcloud release upgrade in AIO?
Current Nexcloud version is 24 but AIO is still 23.
If it takes time to update repository, is it complicated to do it manually?

**Comment by szaimen** [comment, marked as answer]
Hello, there is no way to do it manually. You simply need to wait.
See https://github.com/nextcloud/all-in-one#update-policy

**Reply by agadinmd** [reply]
ok, thanks for the answer
